source/core/parameters_multiphysics.cc

```diff
@@ -30,6 +30,11 @@ Parameters::Multiphysics::declare_parameters(ParameterHandler &prm)
                       Patterns::Bool(),
                       "Free surface calculation <true|false>");
 
+    prm.declare_entry("buoyant force",
```

Comment:
```suggestion
    prm.declare_entry("buoyancy force",
```
I think buoyancy force is the correct naming.